eng/testing/RunnerTemplate.sh

```diff
@@ -1,4 +1,4 @@
-﻿#!/usr/bin/env bash
+#!/usr/bin/env bash
```

Comment:
It seems like you're changing the file permissions, is that intentional?

Reply:
Merged with other PR